docs/source/changelog.rst

```diff
@@ -40,6 +40,7 @@ Bug fixes
   :user:`manics.`
 - Add help text for commandline arguments in :pr:`517` by :user:`yuvipanda`.
 - Fix submodule checkout in :pr:`809` by :user:`davidbrochart`.
+- Fix Nix builds in :pr:`990` by :user:`FRidh`.
```

Comment:
wrong release

Reply:
This changelog is for a past release so I don't think it makes sense to add to them. What was your thinking here?

Reply:
Didn't check the name or number, just assumed the one at the top is the unreleased one, considering project often have that.

I tried following the guidelines here, but its actually not possible to follow them regarding the changelog part because there is no section for them.

Reply:
Sorry about the misleading instructions. My recommendation would be to remove this line.

Or if you have the time start a new section for the next release. Though I am undecided if we will keep having manually made change logs as we are moving towards automatically tagging and releasing repo2docker once a month.